process_pr.py

```diff
@@ -1239,8 +1253,12 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
         elif re.match(r"^\s*" + REGEX_IGNORE_COMMIT_COUNT + r"\s*$", first_line):
             comment_emoji = "-1"
             if commenter in CMSSW_ISSUES_TRACKERS:
-                ok_too_many_commits = True
-                comment_emoji = "+1"
+                if pr.commits < TOO_MANY_COMMITS_FAIL_THRESHOLD:
+                    ok_too_many_commits = True
+                    comment_emoji = "+1"
+                else:
+                    ok_too_many_commits = False
+                    comment_emoji = "-1"
```

Comment:
so this block can be simplify to
```
ok_too_many_commits = pr.commits < TOO_MANY_COMMITS_FAIL_THRESHOLD
comment_emoji = "+1" if ok_too_many_commits else "-1"
```
right?

Reply:
Yes. We can also always set emoji to `+1` if we want to keep `-1` as indication of missing permission or wrong syntax.